<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="clockCheckBox">
            <property name="text">
             <string>&amp;Show current time in full screen mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="menuBarCheckBox">
            <property name="text">
             <string>&amp;Hide menu bar in full screen mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="unbreakableSpaceCheckBox">
            <property name="text">
             <string>Show &amp;Unbreakable Space</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Interface style</string>
              </property>
              <property name="buddy">
               <cstring>cornersComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cornersComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>File Saving</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="autoSaveCheckBox">
            <property name="text">
             <string>&amp;Auto save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backupCheckBox">
            <property name="text">
             <string>&amp;Backup file on save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openDraftDirButton">
            <property name="text">
             <string>&amp;View untitled drafts…</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentBackupFolderText">
            <property name="text">
             <string>&amp;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectBackupFolderButton">
            <property name="text">
             <string>&amp;Select Backup Folder Location…</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Session</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="rememberHistoryCheckBox">
            <property name="text">
             <string>&amp;Remember recent files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="restoreSessionCheckBox">
            <property name="text">
             <string>&amp;Open last file on startup</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Editor">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Tabulation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="spacesCheckBox">
            <property name="text">
             <string>&amp;Insert spaces for tabs</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Tabulation width</string>
              </property>
              <property name="buddy">
               <cstring>tabWidthInput</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tabWidthInput">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>8</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Styling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Focus mode</string>
              </property>
              <property name="buddy">
               <cstring>focusModeCombo</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="focusModeCombo"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Editor width</string>
              </property>
              <property name="buddy">
               <cstring>editorWidthCombo</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="editorWidthCombo"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Blockquote style</string>
              </property>
              <property name="buddy">
               <cstring>blockquoteStyleCombo</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="blockquoteStyleCombo"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Emphasis style</string>
              </property>
              <property name="buddy">
               <cstring>underlineCombo</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="underlineCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="largeHeadingsCheckBox">
            <property name="text">
             <string>&amp;Use large headings</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Typing</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="cycleBulletPointsCheckBox">
            <property name="text">
             <string>C&amp;ycle bullet point markers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoMatchCheckBox">
            <property name="text">
             <string>&amp;Automatically match characters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="matchedCharsButton">
            <property name="text">
             <string>Customize &amp;matched characters…</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>774</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>876</x>
     <y>780</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
